smbios_validation

```diff
@@ -32,7 +32,7 @@ flags.DEFINE_string('file', '', 'dmidecode dump file path')
 def print_source_code_header():
```

Comment:
@ballle98 you should add `Dell` copyright, now that you touched this file...

no need to keep ling license, we can use short SPDX format like this:
```
# SPDX-License-Identifier: Apache-2.0
# Copyright (c) 2019 Google LLC
# Copyright (c) 2022 Dell Inc, or its subsidiaries.
```